### ISSUE TITLE
LiveMedia fails to boot properly in QEMU/KVM VM

### ISSUE THREAD

**Issue by polarathene** [manjaro/packages-core#60]
Not sure what the correct repo is for this, unable to write a forum post about it as no matter how many times I've tried to get the verification e-mail(hotmail/outlook account) I fail to receive it(including spam/junk folder). It seems other users(some posted on manjaro subreddit) about this problem, outlook accounts don't receive the e-mails forums are sending at all.

I've been using QEMU/KVM via virt-manager to install and run VMs For some reason Manjaro boots but fails to get passed `[ OK ] Started TLP system startup/shutdown.` during preventing me from installing the various spins I've tried. Other distro's are fine so I'm not sure what's going on with Manjaro(my host is Manjaro KDE if that matters).

Any information I can provide to figure this out? I'd really like to use Manjaro via VMs managed by virt-manager(Q35 chipset).

**Comment by polarathene** [comment]
Seems to be due to running the VM setup with UEFI instead of BIOS. When using BIOS I'm able to boot the Live Media properly, I noticed some errors that seem to point the issue to be related to SDDM. My host system installed with UEFI so I'm not sure why Manjaro has trouble with it via VM, the other distro's I tried do not.

I've filed an issue at SDDM [here](https://github.com/sddm/sddm/issues/753).

**Comment by polarathene** [comment]
Issue was due to Manjaro MHWD falling back to creating an xorg conf with vesa if an appropriate graphics vendor could not be detected. vesa is not available on UEFI with these graphic modes thus X crashes.

I have filed two issues on the MHWD repo, [let X handle detection when MHWD fails](https://github.com/manjaro/mhwd/issues/31) instead of falling back to vesa. And for VGA(bochs_drm) driver, Plymouth use requires DRM drivers in initrd else prevents displaying VTs, [removing framebuffer support from Plymouth](https://github.com/manjaro/mhwd/issues/32) is said to be a solution. I'm unable to verify if this is required by fixing the xorg conf as I cannot switch TTY due to the bug to make such a change.

I would greatly appreciate a response if the team will consider addressing these issues or not.
